core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -50,7 +50,7 @@ public class TxReader implements Closeable, Mutable {
     protected int symbolColumnCount;
     protected long truncateVersion;
     protected long dataVersion;
-    protected long structureVersion;
+    protected volatile long structureVersion;
```

Comment:
There is a single writer thread, so it doesn't matter. But I've changed the type to `AtomicLong` to avoid confusion. See c7fd0c6e2e1073ba9eec03d462e47a07db8836e3